### DISCUSSION TITLE
SuperSlicer 2.4 keeps asking to save empty platter

### DISCUSSION THREAD

**Discussion by zogthegreat** [supermerill/SuperSlicer#2768] [Q&A] [answered]
Hi everyone!

I'm trying out SuperSlicer 2.4 "Fishy edition". I like the improvements, but I keep having a problem with it when I try to close the program. Every time I try to close SuperSlicer, I get a message 
The platter is empty. Do you want to save the project?

[https://imgur.com/a/gK18yHv](url)

If I choose 'No" then the program won't quit. If I choose "Yes", then SuperSlicer creates a "Untitled.3mf" file. I've dug around in the preferences, but I can't find a way to turn this off. 

Any suggestions?

Thanks!

zog

**Comment by supermerill** [comment, marked as answer]
i'm currently fixing that. Wait the next release.
Last nightly may already have the fix.

**Reply by neophyl** [reply]
Also this is one issue that has also been raised on their github as it does the same.  I suspect they are also looking at some form of solution too.

**Reply by supermerill** [reply]
There is the project check and then the preset check.
I planned to merge them into one window, but it takes a fait amount of time to create a sensible solution.
If prusa will change it for 2.5, then the best short-term solution is to disable the popup with the preferences.

**Comment by zogthegreat** [comment]
Thanks for the reply @supermerill 

(chuckle) I thought it was a PEBKAC!
